### DISCUSSION TITLE
Changelog - new version template

### DISCUSSION THREAD

**Discussion by vhirtham** [BAMWelDX/weldx#422] [Guidelines]
I just gathered most of the fields that we used so far in our changelog. To stay consistent, just copy this to the top of the changelog in the first PR after a new release. Fields without entries can be deleted during the release of a new version.

~~~ rst
********************
 X.X.X (unreleased)
********************

added
=====

removed
=======

changes
=======

fixes
=====

documentation
=============

ASDF
====

deprecations
============

dependencies
============

~~~

**Comment by marscher** [comment]
nice template! Thanks

**Comment by vhirtham** [comment]
Updated to `rst`
